vaults/README.md

```diff
@@ -1,53 +1,64 @@
-# Harshicorp Vault (Bank-Vaults)
+<H1>Harshicorp Vault (Bank-Vaults)</H1>
```

Comment:
Yes but I put <H1> to avoid the titile `Harshicorp Vault (Bank-Vaults)` appearing in the table of content and simplify the numbering (I use an automatic table of content and section numbering with VSCode). Otherwise all the table of content would be auto-prefixed `1.`